Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -212,7 +224,7 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
         if let interval {
             let firstExpirationDate = Date().addingTimeInterval(interval)
 
-            os_log("Setting key validity interval to %{public}@ seconds (next expiration date %{public}@)",
```

Comment:
The "next expiration date" bit was misleading as this method is not doing anything to change the next expiration date.